components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -435,6 +435,7 @@ def pickFirstSignature(condition, filterLambda):
                     # large enough that we can examine this argument.
                     info = getJSToNativeConversionInfo(
                         type, descriptor, failureCode="break;", isDefinitelyObject=True)
+                    assert not info.needsRooting
```

Comment:
Actually, a more immediately applicable example is WebSocket::send with ArrayBuffer/ArrayBufferView overloads.